packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -1121,14 +1161,18 @@ - (BOOL)setupWriterForPath:(NSString *)path {
     AudioChannelLayout acl;
     bzero(&acl, sizeof(acl));
     acl.mChannelLayoutTag = kAudioChannelLayoutTag_Mono;
-    NSDictionary *audioOutputSettings = nil;
-    // Both type of audio inputs causes output video file to be corrupted.
-    audioOutputSettings = @{
+    NSMutableDictionary *audioOutputSettings = [@{
       AVFormatIDKey : [NSNumber numberWithInt:kAudioFormatMPEG4AAC],
       AVSampleRateKey : [NSNumber numberWithFloat:44100.0],
       AVNumberOfChannelsKey : [NSNumber numberWithInt:1],
       AVChannelLayoutKey : [NSData dataWithBytes:&acl length:sizeof(acl)],
-    };
+    } mutableCopy];
+
+    if (_audioBitrate) {
+      // Both type of audio inputs causes output video file to be corrupted.
```

Comment:
not sure if i follow this. can you elaborate more?

Reply:
Sorry, just garbaged comment. Removed.